src/modelframe.jl

```diff
@@ -16,26 +16,27 @@ then creates the necessary contrasts matrices and stores the results.
 ModelFrame(f::Formula, df::AbstractDataFrame; contrasts::Dict = Dict())
 ModelFrame(ex::Expr, d::AbstractDataFrame; contrasts::Dict = Dict())
 ModelFrame(terms::Terms, df::AbstractDataFrame; contrasts::Dict = Dict())
-# Internal constructors:
+# Inner constructors:
 ModelFrame(df::AbstractDataFrame, terms::Terms, missing::BitArray)
 ModelFrame(df::AbstractDataFrame, terms::Terms, missing::BitArray, contrasts::Dict{Symbol, ContrastsMatrix})
 ```
 
 **Arguments**
 
-* `f`: Formula whose left hand side is the *response* and right hand side are
-  the predictors.
-* `df`: The data being modeled.  This is used at this stage to determine which
-  variables are categorical, and otherwise held for [`ModelMatrix`](@ref).
-* `contrasts`: An optional Dict of contrast codings for each categorical
+* `f::Formula`: Formula whose left hand side is the *response* and right hand
+  side are the *predictors*.
+* `df::AbstractDataFrame`: The data being modeled.  This is used at this stage
+  to determine which variables are categorical, and otherwise held for
+  [`ModelMatrix`](@ref).
+* `contrasts::Dict`: An optional Dict of contrast codings for each categorical
```

Comment:
"contrast coding schemes" would be better (IIRC sometimes "contrast coding" refers to non-standard/complex contrast schemes).